KIImagePager/KIImagePager/KIImagePager.m

```diff
@@ -258,6 +258,7 @@ - (void) loadData
         }
         [_scrollView addSubview:blankImage];
     }
+    self.hidePageControlForSinglePages = self.hidePageControlForSinglePages;
```

Comment:
We should rather extract the logics from the setter than setting the property to itself.
Nobody will know what this code was for and it will probably result in unexpected behaviour during further improvements.